lib/membrane_rtc_engine/engine.ex

```diff
@@ -197,11 +107,115 @@ defmodule Membrane.RTC.Engine do
     {:noreply, socket}
   end
   ```
+
+  ## Peers
+
+  Each peer represents some user that can possess some metadata.
+  A Peer can be added in two ways:
+  * by sending proper Media Event from a Client Library
+  * using `add_peer/3`
+
+  Adding a peer will cause RTC Engine to emit Media Event which will notify connected clients about new peer.
+
+  Each Endpoint you want it to send some tracks from RTC Engine to Membrane client library has to be assigned to some peer.
```

Comment:
I'd move this sentence to the chapter about endpoints

Reply:
Actually, I would remove this sentence as it depends on Client Library implementation